crates/hdk/src/hash_path/path.rs

```diff
@@ -291,26 +315,63 @@ impl Path {
     }
 
     /// Touch and list all the links from this anchor to anchors below it.
-    /// Only returns links between anchors, not to other entries that might have their own links.
+    /// Only returns links between paths, not to other entries that might have their own links.
     pub fn children(&self) -> ExternResult<Vec<holochain_zome_types::link::Link>> {
         Self::ensure(self)?;
         let mut unwrapped = get_links(
-            self.hash()?,
-            Some(holochain_zome_types::link::LinkTag::new(NAME)),
+            self.path_entry_hash()?,
+            Some(holochain_zome_types::link::LinkTag::new(DHT_PREFIX)),
```

Comment:
Isn't it more likely to have collisions at this tag now that the tag is just `0`? Or is it enforced that a tag can't be `0`?

Reply:
@maackle sure, from a base that is a hash of a hash, so if you want to put links on a path entry and not have them show up as children (a pretty deliberate thing) then prefix it with `1`